cukinia

```diff
@@ -196,6 +206,26 @@ _cukinia_http_request()
 	wget -q -O /dev/null "$url"
 }
 
+# _cukinia_mount: check for a specific mount point
+# arg1: the mount source
+# arg2: the mount destination
+# arg@: (optinal) a list of options to look for
```

Comment:
this one only works if the options are specified in the exact same order as mount displays them, and I don't think there's any warranty on that order. So I wouldn't add this last option.

Note there's an alternative in reading /proc/mounts:  ($1 == "$device" && $2 == "$mountpoint")